README.md

```diff
@@ -211,3 +211,38 @@ end
 s = [MyType(rand(), a=1, b=2) for i in 1:10]
 StructArray(s)
 ```
+
+## Advanced: mutate-or-widen style accumulation
+
+StructArrays provides `grow_to_structarray!(dest, src)` which works like `append!(dest, src)` if `dest` can contain all element types in `src` iterator; i.e., it _mutates_ `dest` in-place:
```

Comment:
In practice this can now be `StructArrays.append!!` I imagine, and it is the only function that you need to call from BangBang, right?

Reply:
Yes.